packages/admin-ui/src/tag/tag.style.ts

```diff
@@ -0,0 +1,37 @@
+import { colors, style, styleVariants } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+function customPalette(color: string) {
+  return {
+    bg: get(colors, `${color}10`, ''),
+    color: '$primary',
+  }
+}
+
+export const baseline = style({
+  display: 'inline-flex',
+  alignItems: 'center',
+  borderRadius: '100px',
+  text: '$detail',
+  color: '$primary',
+  border: 'none',
+  height: '1.5rem',
+  paddingX: '0.75rem',
+  outlineColor: (theme) => theme.bg.primary,
```

Comment:
I would replace this for:
```jsx
import { theme } from '@vtex/admin-ui-core'
...
  outlineColor: get(theme, 'bg.primary', 'bg.primary')
```

Or

```jsx
  outlineColor: (theme) => get(theme, 'bg.primary', 'bg.primary')
```

Reply:
Why are the approaches above preferable?

Reply:
Because we allow custom themes. Using `get` is a safer way of accessing the theme value, the other way can lead to accessing an undefined value

Reply:
Got it! Thanks for the explanation, Lucas!